### PR TITLE
delete unused param parentComponentProps from jsxAttributeToValue

### PR DESCRIPTION
**Pull request by balazsbajorics**
super mini PR! I noticed parentComponentProps was not used anymore

### CODE REVIEW

File: editor/src/utils/canvas-react-utils.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function applyUIDMonkeyPatch(): void {
   }
 }
 
-function getDisplayName(type: any) {
+export function getDisplayName(type: any) {
```
Comment (Rheeseyb) [inline editor/src/utils/canvas-react-utils.ts:25]: Was this change intended?

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/utils/canvas-react-utils.ts (hunk 1 above)
Comment (balazsbajorics) [reply editor/src/utils/canvas-react-utils.ts:25]: nope

**Comment by balazsbajorics** [comment]
I am merging this because of the very small impact
